common/envOverwrite/overwriter.go

```diff
@@ -2,33 +2,58 @@ package envOverwrite
 
 import "strings"
 
-var envValues = map[string]string{
-	"NODE_OPTIONS": "--require /var/odigos/nodejs/autoinstrumentation.js",
+// EnvValues is a map of environment variables odigos uses for various languages and goals.
+// The key is the environment variable name and the value is the value to be set or appended
+// to the environment variable. We need to make sure that in case any of these environment
+// variables is already set, we append the value to it instead of overwriting it.
+var EnvValues = map[string]struct{
+	Value string
+	Delim string
+}{
+	"NODE_OPTIONS": {
+		Value: "--require /var/odigos/nodejs/autoinstrumentation.js",
+		Delim: " ",
+	},
+	"PYTHONPATH": {
+		Value: "/var/odigos/python/opentelemetry/instrumentation/auto_instrumentation:/var/odigos/python",
+		Delim: ":",
+	},
 }
 
-func ShouldOverwrite(envName string) bool {
-	_, ok := envValues[envName]
+func ShouldPatch(envName string, value string) bool {
+	_, ok := EnvValues[envName]
 	return ok
 }
 
-func Patch(envName string, currentVal string) string {
-	val, exists := envValues[envName]
-	if !exists {
-		return ""
+func ShouldRevert(envName string, value string) bool {
+	_, ok := EnvValues[envName]
+	if !ok {
+		// We don't care about this environment variable
+		return false
 	}
 
-	if currentVal == "" {
-		return val
+	if !strings.Contains(value, EnvValues[envName].Value) {
```

Comment:
nit: instead of `EnvValues[envName]` again you can use the value returned 4 lines above

Reply:
Fixed